src/server/game/Spells/SpellInfoCorrections.cpp

```diff
@@ -4459,6 +4459,18 @@ void SpellMgr::LoadSpellInfoCorrections()
         }
     });
 
+    // Self Visual - Sleep Until Cancelled(DND)
+    ApplySpellFix({ 14915 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->AuraInterruptFlags &= ~AURA_INTERRUPT_FLAG_NOT_SEATED;
+    });
+
+     // Cleansing Totem, Healing Stream Totem, Mana Tide Totem
+    ApplySpellFix({ 8171,52025, 52041, 52042, 52046, 52047, 52048, 52049, 52050, 58759, 58760, 58761, 39610, 39609 }, [](SpellInfo* spellInfo)
+    {
+        spellInfo->AttributesEx2 |= SPELL_ATTR2_IGNORE_LINE_OF_SIGHT;
+    });
```

Comment:
can we move these changes to db ideally? (I do not want to block this PR for this, just wondering)

Reply:
Hm? These come from merge...

Reply:
I see, but I was wondering that, because I usually see this ApplySpellFix, but I think that we could use spell_dbc table for it
anyways, it's a bit off-topic

Reply:
spell_dbc table should not be used for this kind of change as it obfuscates the custom edits of spells

Reply:
spell_dbc is strictly for server-side spells only.

Reply:
:ok_hand: